src/Components/Web.JS/src/Rendering/Renderer.ts

```diff
@@ -13,6 +13,8 @@ interface BrowserRendererRegistry {
 }
 const browserRenderers: BrowserRendererRegistry = {};
 let shouldResetScrollAfterNextBatch = false;
+let elementToScrollTo : string | null = null;
+let scrollToElementTimeout : NodeJS.Timeout;
```

Comment:
Could we avoid referencing `NodeJS` types here? The reason is this code is not going to run in Node, so if we use Node types it's quite difficult to reason about whether or not it will work correctly in a browser.

I know in this case that it would work because browser also support `setTimeout` but in browsers the result type for that is just `number` rather than `NodeJS.Timeout`.